bot.ts

```diff
@@ -10,10 +10,25 @@ import {
 	InteractionsClient,
 	slash,
 	SlashCommandPartial,
-} from "https://deno.land/x/harmony@v2.6.0/mod.ts";
-import { serve, Status } from "https://deno.land/std@0.125.0/http/mod.ts";
+} from "https://deno.land/x/harmony@v2.9.1/mod.ts";
+import { STATUS_CODE } from "https://deno.land/std@0.224.0/http/mod.ts";
```

Comment:
If we don't use `serve` anymore I think it's entirely resonable to drop STATUS_CODE here as well, and drop a whole dependency. Up to you

Reply:
`serve` was deprecated and replaced with `Deno.serve`. So, there's still code about serving stuff

Reply:
Yep, but in that case you can remove the whole line `import { STATUS_CODE } from "https://deno.land/std@0.224.0/http/mod.ts";`, and either hardcode 401/404 or add an object like 

```ts
const STATUS_CODE = {
  Unauthorized: 401,
  NotFound: 404,
}
```

Reply:
I've now made it hard coded